### PR TITLE
Rental

### PR DESCRIPTION
**Pull request by Aditijainnn**
Closes: 
 #579

- **Title** : property rental website
- **Name:** :  Aditi
- **Idenitfy yourself:** :  GSSOC

### Screenshots 📷

image

### Type of change ☑️

What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### Checklist: ☑️

- [x] My code follows the [Contributing Guidelines](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CONTRIBUTING.md) & [Code of Conduct](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CODE_OF_CONDUCT.md) of this project.
- [x] This PR does not contain plagiarized content.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly wherever it was hard to understand.
- [x] My changes generate no new warnings.

### CODE REVIEW

**Comment by Avdhesh-Varshney** [comment]
Review your pr by your own and follow the guidelines carefully.

**Comment by Aditijainnn** [comment]
> Review your pr by your own and follow the guidelines carefully.

yes reviewed

**Comment by Aditijainnn** [comment]
@Avdhesh-Varshney please merge my pr

File: Typescript-Projects/Advanced/Rental-Website/README.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+You can start editing the page by modifying `app/page.tsx`. The page auto-updates as you edit the file.
+
+This project uses [`next/font`](https://nextjs.org/docs/basic-features/font-optimization) to automatically optimize and load Inter, a custom Google Font.
+
+<!-- -------------------------------------------------------------------------------------------------------------- -->
+
+## :zap: Screenshots 📸
+<!-- add the screenshot of the project (Mandatory) -->
+
+<img src='./Countdown-Timer.webp'>
```
Comment (Avdhesh-Varshney) [inline Typescript-Projects/Advanced/Rental-Website/README.md:47]: Where is this image?
- You've to follow the guidelines of ss as mentioned in your earlier pr.

**Review by Avdhesh-Varshney — CHANGES_REQUESTED** [review]
You've committed your earlier project also in this project.
Either delete the un-necessary project or closed this pr.

**Pull request closed without merging**